calculator/style.css

```diff
@@ -0,0 +1,66 @@
+* {
+  margin: 0px;
+  padding: 0px;
+  box-sizing: border-box;
+}
+
+.container {
+  position: absolute;
+  top: calc(50% - 250px);
+  left: calc(50% - 200px);
+}
+
+.calculator {
+  width: 400px;
+  height: 500px;
+  background: #1dca34;
+  border-radius: 10%;
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  box-sizing: border-box;
+}
+
+.calculator-display {
+  width: 80%;
+  height: 10%;
```

Comment:
Just a note: If you know exactly the width and height for you the input block and you understand that this block will have the same parameters for desktop and mobile view.  I think will be better to use specific units of measure.

Reply:
Thanks for the advice, I'll fix it